Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -26,6 +28,8 @@ protocol HomePanel: class {
 
 @objc
 protocol HomePanelDelegate: class {
+    func homePanelDidRequestToSignIn(homePanel: HomePanel)
```

Comment:
Only the `HomePanelViewController` implement the delegate right now, which implements the whole thing.

I do think those two methods could go in a separate delegate. I tried that initially but I ran into code that was just too hard to deal with and would require a major refactoring.